release/package.json

```diff
@@ -1273,38 +1261,32 @@
 				"FSharp.simplifyNameAnalyzer": {
 					"type": "boolean",
 					"default": true,
-					"description": "Enables detection of symbols usages that can be simplified"
+					"description": "Enables detection of cases when names of functions and values can be simplified"
 				},
 				"FSharp.fsiExtraParameters": {
 					"type": "array",
 					"default": [],
-					"description": "Allows to pass extra parameters to FSI process"
+					"description": "An array of additional command line parameters to pass to FSI when it is started. See [the documentation](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/fsharp-interactive-options) for an exhaustive list."
 				},
 				"FSharp.saveOnSendLastSelection": {
 					"type": "boolean",
 					"default": false,
-					"description": "Save Current file before send LastSelection to FSI"
-				},
-				"FSharp.msbuildLocation": {
```

Comment:
we don't support .net fx msbuild anymore, the code in the msbuild module wasn't ever using this